boards/common/atmega/board.c

```diff
@@ -23,10 +23,25 @@
 #include "irq.h"
 #include "periph/gpio.h"
 
+#ifndef CPU_ATMEGA_CLK_SCALE_INIT
+#define CPU_ATMEGA_CLK_SCALE_INIT    CPU_ATMEGA_CLK_SCALE_DIV1
+#endif
+
 void led_init(void);
 
+static inline void set_prescaler(void)
```

Comment:
This one into `cpu/atmega_common/include/cpu.h` as `static inline void atmega_set_prescaler(void)`.

Reply:
Fixed, but also takes the clock scale as an argument, which means that cpu.h doesn't need another include.